packages/sdk/src/clients/registry/client.ts

```diff
@@ -1,18 +1,14 @@
-import { EthersContracts } from "@uma/core";
-import assert from "assert";
-import RegistryArtifacts from "@uma/core/build/contracts/Registry.json";
+import { RegistryEthers, RegistryEthers__factory, getAddress as getAddressForContract } from "@uma/contracts-node";
 import type { SignerOrProvider, GetEventType } from "../..";
 import { Event } from "ethers";
 
-export type Instance = EthersContracts.Registry;
-const Factory = EthersContracts.Registry__factory;
+export type Instance = RegistryEthers;
+const Factory = RegistryEthers__factory;
 
-export type Network = keyof typeof RegistryArtifacts.networks;
+export type Network = string | number;
```

Comment:
maybe theres no way to do this now, but before, typescript would complain if you used the wrong number. this is really minor though

Reply:
Oh really sorry -- I didn't mean to merge before replying to this. The package just takes a normal number rather than a const type. I _think_ the reason I didn't do this in the contracts-node package is because in tests, you can connect to a local node that might have a different network id that's not hardcoded into the package. I'm guessing the sdk always assumes that it's being run on production networks?